docs/advanced/scf/soc.md

```diff
@@ -0,0 +1,25 @@
+## SOC Effects
+### SOC 
+`lspinorb` is used for control whether or not SOC(spin-orbit coupling) effects should be considered.
+
+Both `basis_type=pw` and `basis_type=lcao` support `scf` and `nscf` calculation with SOC effects.
+
+Atomic forces and cell stresses can not calculated with SOC effects until now. 
```

Comment:
In my humble opinion, "until now" means you can not do something before, but you can do it from now on. I guess this is not what you mean.
```suggestion
Atomic forces and cell stresses can not be calculated with SOC effects yet. 
```